README.md

```diff
@@ -0,0 +1,36 @@
+# Gatekeeper
+
+This application is responsible for opening and closing the front gate.
+
+## Context
+
+Gatekeeper is a raspberry pi (hostname `zuul`) that is responsible for opening the front gate for guest and members. The pi is used to control the gate remote control and uses a Sim800 module to receive calls
+
+## Usage
+
+The gatekeeper application uses a whitelist in the following format:
+
+    +123456 owner
+
+The first element is the phone number in international notation without spaces or other punctuation, the second element is a free text field describing the owner of the phone number.
+
+The application can be deployed using the included systemd service.
+
+To run the application without systemd, use the following command:
+
+    python3 main.py -d ./whitelist -v --no-journald -m mqtt
+
+## MQTT
+
+Gatekeeper can also be monitored and controlled using MQTT.
+
+Gatekeeper listens to commands on the `hsg/gatekeeper/cmd` topic. So far it supports the following commands:
+
+* `open`: open the gate
+
+Gatekeeper will also publish events on the `hsg/gatekeeper/*` topics:
+
+* `hsg/gatekeeper/open`: this topic will have a message published with the owner as described in the data file upon a successful call to open the gate.
```

Comment:
it is unclear to me what you mean with "the owner" (i don't believe mqtt messages have owner attributes)

Reply:
owner here, as in, the person who called. Then this could be used as a source for eg flappers.

Reply:
ah, maybe something like "upon a successful call to open the gate, the "owner" field from the whitelist will be published here"